### PR TITLE
7.x silly libvpx

### PR DESCRIPTION
**Pull request by adam-vessey**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ php:
   - 5.4
 branches:
   only:
-    - 7.x
+    - /^7.x/
```
Comment (willtp87) [inline .travis.yml:7]: I think we should change this line in each branch instead.

Comment (adam-vessey) [inline .travis.yml:7]: Can't, has to be in the "default" branch.

Comment (adam-vessey) [inline .travis.yml:7]: This just means that each `7.x` branch will be built... Commits in one branch won't cause a build in another.

Comment (adam-vessey) [inline .travis.yml:7]: Admittedly, perhaps the `.` should be escaped, since it's a regex... Shrug.

Comment (adam-vessey) [inline .travis.yml:7]: Actually, now looking at [the docs](http://docs.travis-ci.com/user/build-configuration/#Specify-branches-to-build), I'm not 100% sure that it _has_ to be in the default branch... Shouldn't really be a problem, anyway... we do this in other repos.

Comment (willtp87) [inline .travis.yml:7]: Perhaps we can pull it out of this request and do some experimenting, treat it as it's own thing.  My curiosity is intensifying but getting the repo running would be good.

Comment (adam-vessey) [inline .travis.yml:7]: Don't want to get out my work laptop (had done this before leaving the office), so if it's to be stripped out (by me, anyway), it's going to wait for Monday.

**Pull request merged**